code/modules/research/designs/misc_designs.dm

```diff
@@ -286,6 +286,26 @@
 	build_path = /obj/item/locator
 	category = list("Equipment")
 	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
+	
+/datum/design/quantum_keycard
+	name = "Quantum Keycard"
+	desc = "Allows for the construction of a quantum keycard."
+	id = "quantum_keycard"
+	build_type = PROTOLATHE
+	materials = list(MAT_GLASS = 500, MAT_METAL = 500, MAT_SILVER = 500, MAT_BLUESPACE = 1000)
+	build_path = /obj/item/quantum_keycard
+	category = list("Equipment")
+	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE | DEPARTMENTAL_FLAG_ENGINEERING
+	
+/datum/design/hand_tele
+	name = "Hand Teleporter"
+	desc = "Allows for the construction of a hand teleporter."
+	id = "hand_tele"
+	build_type = PROTOLATHE
+	materials = list(MAT_GLASS = 500, MAT_METAL = 500, MAT_GOLD = 500, MAT_BLUESPACE = 2000, MAT_DIAMOND = 2000)

```

Comment:
Diamond to 3000 maybe.

Reply:
Nah, would make sense to bump up the bluespace crystal cost instead of the diamond one.

Reply:
Both 2100 would be fine: The threshold is 2000 units per sheet and I'm suggesting that the hand tele should cost just a bit more on paper than 1 sheet.